pipelines/release.yml

```diff
@@ -991,14 +992,25 @@ resources:
   source:
     uri: https://github.com/concourse/concourse.git
     branch: release/((release_minor)).x
+    ignore_paths:
+    - release-notes/
 
 - name: ci
   type: git
-  icon: &git-icon github-circle
+  icon: *git-icon
   source:
     uri: https://github.com/concourse/ci.git
     branch: master
 
+- name: release-notes
+  type: git
+  icon: *git-icon
+  source:
+    uri: https://github.com/concourse/concourse.git
+    branch: master
```

Comment:
is this `master` because we'd write release notes for, say, `5.2.4` on `master`, rather than on the `5.2.x` branch?

Reply:
Exactly. If there's any single place where you'd expect to find all the release notes, that folder on master seems like the natural place. Id rather that than have the release notes scattered among multiple branches, but that's a subjective preference.